catalog/app/utils/workflows.js

```diff
@@ -38,6 +50,7 @@ function parseWorkflow(workflowSlug, workflow, data) {
 }
 
 const parseSuccessor = (url, successor) => ({
+  copyData: successor.copy_data || false,
```

Comment:
```suggestion
  copyData: successor.copy_data != null ? successor.copy_data : COPY_DATA_DEFAULT,
```

Reply:
I'm comparing with undefined

Reply:
ok, `x == null` is basically a shorthand for `x === null || x === undefined`